CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 [a2b58c3d...HEAD](https://github.com/rust-lang/cargo/compare/a2b58c3d...HEAD)
 
 ### Added
+- Added the `path-bases` feature to support paths that resolve relatively to a
+  base specified in the config.
+  [docs](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#path-bases)
+  [#14360](https://github.com/rust-lang/cargo/pull/14360)
```

Comment:
Normally changelogs are handled semi-automatically.  Unsure if a manually written entry will interfere.  @weihanglo ?

If we do keep this, it needs to be in a "Nightly only" section

Reply:
Moved into the "Nightly only" section, but I'm happy to remove this if you'd prefer.

Reply:
The changelog process skips pull requests that are already listed in the file, so should be fine.

We usually combine relevant PRs into one entry, but that should be fine here also.

A risk here is we are not 100% sure which version this PR is going to get into, though that's also fine as we can delete the entry then.

Reply:
I've removed the note: I don't want to block this PR on getting the release note exactly right. I'm happy to discuss the text here or on zulip.